### PR TITLE
Grab bag of mini tests

### PR DESCRIPTION
**Pull request by kshyatt**
Tests of `Tuple`, a few `BigInt` functions, and `contains`.

### CODE REVIEW

File: base/tuple.jl [hunk 1]
```diff
@@ -49,7 +49,7 @@ function front(t::Tuple)
     @_inline_meta
     _front((), t...)
 end
-front(::Tuple{}) = error("Cannot call front on an empty tuple")
+front(::Tuple{}) = throw(ArgumentError("Cannot call front on an empty tuple"))
```
Comment (tkelman) [inline base/tuple.jl]: kind of a boundserror of a sort, isn't it?

File: test/bigint.jl [hunk 2]
```diff
@@ -300,3 +300,14 @@ ndigits(rand(big(-999:999)), big(2)^rand(2:999))
...
 @test_throws InexactError floor(BigInt,Inf)
 @test_throws InexactError ceil(BigInt,Inf)
+
+@test bin(big(3)) == "11"
+@test oct(big(9)) == "11"
+@test oct(-big(9)) == "-11"
+@test hex(big(12)) == "c"
+
+@test isqrt(big(4)) == 2
+@test isqrt(big(5)) == 2
```
Comment (tkelman) [inline test/bigint.jl]: use `===` if possible to also test the type?

Comment (kshyatt) [inline test/bigint.jl]: You mean for the `isqrt`?

Comment (tkelman) [inline test/bigint.jl]: everything where possible

File: base/tuple.jl (hunk 1 above)
Comment (kshyatt) [inline base/tuple.jl]: Can you pass strings to `BoundsError`?

Comment (tkelman) [inline base/tuple.jl]: no, I believe you cannot

Comment (kshyatt) [inline base/tuple.jl]: Then I think keeping it `ArgumentError` makes sense so that we can give people more info about what's wrong.

Comment (KristofferC) [inline base/tuple.jl]: Typically error messages start with lower case if you feel like changing that while you are at it.

File: test/bigint.jl (hunk 2 above)
Comment (ivarne) [inline test/bigint.jl]:
```jlterm
julia> big"2" === big"2"
false
```

Comment (kshyatt) [inline test/bigint.jl]: I was wondering about this too. I think only `==` will work...

**Comment by kshyatt** [comment]
I'm going to merge and come back for the error message if that is ok.

**Pull request merged**